### PR TITLE
UI changes, navigation overhaul, frontpage icon additions

### PR DESCRIPTION
**Pull request by saintsoldierx**
* [x]  Passed [sanity tests](http://bit.ly/sttm-sanity-tests).
* [x]  Ran `npm test` & fixed newly introduced lint errors.
* [x]  Checked console for errors.

<!-- New to markdown? Simply put an 'x' between [ ] to check it. Like [x] this. (No spaces).

### CODE REVIEW

**Comment by saintsoldierx** [comment]
@navdeepsingh please check this out - one thing, your test cases for multiview were giving me issues so i may have removed them.

**Comment by navdeepsingh** [comment]
> @navdeepsingh please check this out - one thing, your test cases for multiview were giving me issues so i may have removed them.

Yeah Snapshot test making it fail, will improve it another time.

File: src/js/components/AddShabadButton.tsx [hunk 1]
```diff
@@ -48,7 +48,8 @@ export const AddShabadButton: React.FC<FCProps> = ({ shabad: { verseId, shabadId
       data-verse={verse}
       onClick={onClick}>
       <AddIcon />
-      Add
+      <div className="show-on-desktop">
```
Comment (navdeepsingh) [inline src/js/components/AddShabadButton.tsx:51]: At homepage search, Autocomplete not works if we add that (it's weird), I will fix it on my end that portion.

Comment (saintsoldierx) [reply src/js/components/AddShabadButton.tsx:51]: Good catch! Yes, please fix on your end then.

**Pull request merged**